docs/docs/user-guide/github-actions.md

```diff
@@ -110,6 +149,60 @@ Congratulations! You have just setup your first custom workflow in a GitHub repo
 
 ![CI Successful build](../images/github-actions-tutorial-success.png "CI Successful Build")
 
+#### Specify the version of CLI for Microsoft 365
+
+Sometimes, you would like to use the next (beta) version or a previous one of the CLI (because of a bug that is currently being addressed or to leverage a new feature to come).
+
+The `CLI for Microsoft 365 Login` action provides an optional input to specify a version tag which can be `latest` (default if not specified), `next` or a specific one (`5.9.0`).
+
+```yaml
+  name: CI
+
+  on: [push]
+
+  jobs:
+    build:
+
+      runs-on: ubuntu-latest
+
+      steps:
+
+      - name: Login to tenant
+        uses: pnp/action-cli-login@v2.2.1
+        with:
+          APP_ID: ${{ secrets.APP_ID }}
+          CERTIFICATE_ENCODED: ${{ secrets.CERTIFICATE_ENCODED }}
+          CERTIFICATE_PASSWORD: ${{ secrets.CERTIFICATE_PASSWORD }}
+          CLI_VERSION: next
+```
+
+#### Specify the tenant to connect to
+
+Depending on the context, you could have to specify the tenant id if you have access to multiple tenants.
+
+The `CLI for Microsoft 365 Login` alows you to indicate the targeting tenant (default `common`). It can be `organization` (if using an Azure AD app which is multitenant) or a tenant id.
```

Comment:
```suggestion
The `CLI for Microsoft 365 Login` allows you to indicate the targeting tenant (default `common`). It can be `organization` (if using an Azure AD app which is multitenant) or a tenant id.
```